packages/create-catalyst/src/utils/clone-catalyst.ts

```diff
@@ -123,7 +132,24 @@ export const cloneCatalyst = async ({
 
   tsConfigJson.compilerOptions.paths['@bigcommerce/components/*'] = ['./components/ui/*'];
 
+  if (!includeFunctionalTests) {
+    tsConfigJson.include = tsConfigJson.include?.filter(
+      (include) => !['tests/**/*', 'playwright.config.ts'].includes(include),
+    );
+  }
+
   writeJsonSync(join(projectDir, 'tsconfig.json'), tsConfigJson, { spaces: 2 });
 
+  if (!includeFunctionalTests) {
+    const eslint = (await readFile(join(projectDir, '.eslintrc.cjs'), { encoding: 'utf-8' }))
+      .replace(/['"]\/playwright-report\/\*\*['"],?/, '')
+      .replace(/['"]\/test-results\/\*\*['"],?/, '');
```

Comment:
Using regex instead as it could be multi-lined. Pretty sure `eslint --fix` will take care of the formatting. If not we can always revisit it and do some cleanup here.

Reply:
Here were the regex test cases I used:
![Screenshot 2024-04-23 at 11 28 52](https://github.com/bigcommerce/catalyst/assets/10539418/190fbc17-fc1a-4cd9-aad3-6828b10e3831)